### ISSUE TITLE
Need Help on writing Javascript code

### ISSUE THREAD

**Issue by MohamedMafaz** [MohamedMafaz/Propoplex#33]
Hey folks!
Contributor need to write a code that get prompt from the user as input and need to print the output in html file with help of Google Bard API. If you're interested to work on this issue. Just comment below your email id. And I'll send the API Key only for you

**Comment by okondev2021** [comment]
Hi, i am interested my email address --> sojapril16@gmail.com. thanks

**Comment by MohamedMafaz** [comment]
Email Sent.

**Comment by shauryaojaswa** [comment]
hey I am interested in this project

**Comment by MohamedMafaz** [comment]
Hey @shauryaojaswa 
@okondev2021 already assigned to this issue.  You may take a look at other issue or open some issues

Thanks,
Mohamed Mafaz PR,
Propoplex.

**Comment by MohamedMafaz** [comment]
hey @shauryaojaswa,
I have created a file called script.mjs.
when I run ```node script.jsx``` it works fine. You need to do:
- Need to declare variables from the user input
- The code need to generate proposals from website command
- Need to show the output in the <p tag in HTML.

If you completed it and open a PR. I can assign this issue & merge it.

**Comment by MohamedMafaz** [comment]
Hey! Any Updates

**Comment by okondev2021** [comment]
Trying to get some things right. Using bard

**Comment by okondev2021** [comment]
Also do you have the documentation, if you do please send

**Comment by okondev2021** [comment]
Also did you get the json file, i am new to using Google bard api. If you
have any resources i can use please let me know.

**Comment by okondev2021** [comment]
Or we can use chatgpt

**Comment by MohamedMafaz** [comment]
Hey @okondev2021 
Try using Bard. If it seems too hard to integrating Bard then go with ChatGPT API

**Comment by okondev2021** [comment]
Tried using chatgpt api, but its advisable to use a server side language to
protect the api key plus its not completely free.
If you have the documentation on how to use google bards api please send so
i can work with it.
I noticed you plan on using flask
I am still learning. Thank you

**Comment by MohamedMafaz** [comment]
Hey @okondev2021 
Sorry for the delay. You can check here https://www.npmjs.com/package/bard-ai

**Comment by MohamedMafaz** [comment]
Hey @okondev2021 
Any Updates?

**Comment by okondev2021** [comment]
Yes, tested the response with bard and its working well. Have been busy
with school work sorry for the delay.

**Comment by MohamedMafaz** [comment]
Okay. I'm also school student So, I can get your pain🥲
Once you're free please submit a PR
